docs/data-sources/organization_members.md

```diff
@@ -0,0 +1,23 @@
+---
+page_tile: "Splunk Observability Cloud - signalfx_organization_members
+description: |-
+    Allows for members to be queried and used as part of other resources. Requires the supplied token to have Admin priviledges.
+---
+
+# Data Source: signalfx_organization_members
+
+Allows for members to be queried and used as part of other resources. Requires the supplied token to have Admin priviledges.
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `emails` (List of String) A list of email address that can be matched against existing users of the organization.
```

Comment:
I'd so something like this:

- `emails`. List of strings. A list of email address that can be matched against existing users of the organization.

Reply:
With the brackets and punctuation, this is handled as part of the auto generation of the terraform provided docs generation. 

What I imagine the internal template looks like is:
```template
`{{ field }}` ({{ type }}) {{ description }}
```  

More of less, this is the expected standard format for terraform docs as provided by terraform.